frontend-web/webclient/app/Project/Subprojects.tsx

```diff
@@ -266,7 +266,8 @@ const Subprojects: React.FunctionComponent = () => {
                                     <Heading.h4>No resources attached to project. Contact project PI</Heading.h4>
                                 } </>
                                 : <>{mainWallets.map((w, i) =>
-                                    <SelectableWallet
+                                    isQuotaSupported(w.wallet.paysFor) ?
```

Comment:
This is a fine solution. Otherwise, you could have written:
```
{mainWallets.map((w, i) => {
    const quotaSupport = isQuotaSupported(w.wallet.paysFor);
    return (<SelectableWallet 
     /*
```
This isn't better, just a different approach.